backend/src/user/user.controller.ts

```diff
@@ -24,7 +24,7 @@ import { UserService } from "./user.service";
 export class UserController {
   constructor(private readonly userService: UserService) { }
 
-  @Post()
+  @Post("create")
```

Comment:
Actually, closer to REST/CRUD would be having all the endpoints add "/user" and deciding what to do based on the HTTP Method. And then another endpoint "users", only supporting get, which lists all users. If this can be implemented quickly, I would really fancy that, as this is probably the example all other implementations will be based upon.

Reply:
Most RESTful would probably be:

GET user/ - returns all users
POST user/- creates a new user, returns the created on
GET user/:id - returns a specific user
PATCH user/:id - updates a specific user
DELETE user/:id - deletes a specific user

Reply:
Update looks really good, I'm just not approving because the tests will fail. We have a little problem with absolute imports in the tests. You will need to use relative imports for now. Didn't have the time to look into it so far.